### ISSUE TITLE
[DateInput] Allow changing AM/PM by pressing a/p

### ISSUE THREAD

**Issue by PeterStaev** [telerik/kendo-angular#2053]
## I'm submitting a...

* Suggestion for improvement  

## Current behavior

Currently when the user enters the AM/PM zone of the dateinput/timerpicker the only way to switch between the values is using the arrows. 

## Expected behavior

It will be good also accept pressing `a` or `p` to change the value. 

## Environment

Package versions:

```

+-- @angular-devkit/build-angular@0.10.7
+-- @angular/animations@7.0.4
+-- @angular/cli@7.0.7
+-- UNMET PEER DEPENDENCY @angular/common@7.0.4
+-- @angular/compiler@7.0.4
+-- @angular/compiler-cli@7.0.4
+-- UNMET PEER DEPENDENCY @angular/core@7.0.4
+-- UNMET PEER DEPENDENCY @angular/forms@7.0.4
+-- @angular/http@7.0.4
+-- @angular/language-service@7.0.4
+-- @angular/platform-browser@7.0.4
+-- @angular/platform-browser-dynamic@7.0.4
+-- @angular/router@7.0.4
+-- @angularclass/hmr@2.1.3
+-- @aspnet/signalr@1.1.0
+-- @auth0/angular-jwt@2.1.0
+-- @fortawesome/angular-fontawesome@0.3.0
+-- @fortawesome/fontawesome-svg-core@1.2.13
+-- @fortawesome/pro-regular-svg-icons@5.7.0
+-- @fortawesome/pro-solid-svg-icons@5.7.0
+-- @ng-bootstrap/ng-bootstrap@4.0.2
+-- @progress/kendo-angular-buttons@4.3.2
+-- @progress/kendo-angular-dateinputs@3.5.4
+-- @progress/kendo-angular-dialog@3.10.1
+-- @progress/kendo-angular-dropdowns@3.4.2
+-- @progress/kendo-angular-editor@0.2.0
+-- @progress/kendo-angular-excel-export@2.3.0
+-- @progress/kendo-angular-grid@3.12.0
+-- UNMET PEER DEPENDENCY @progress/kendo-angular-inputs@4.0.0
+-- @progress/kendo-angular-intl@1.6.1
+-- @progress/kendo-angular-l10n@1.3.0
+-- @progress/kendo-angular-layout@3.2.0
+-- @progress/kendo-angular-notification@1.2.0
+-- @progress/kendo-angular-popup@2.5.0
+-- UNMET PEER DEPENDENCY @progress/kendo-angular-toolbar@1.0.0
+-- @progress/kendo-data-query@1.5.0
+-- @progress/kendo-drawing@1.5.8
+-- @progress/kendo-theme-default@3.2.0
+-- @progress/kendo-ui@2019.1.115
+-- @progress/telerik-angular-report-viewer@6.19.123
+-- @tangra/angular-interceptors@1.3.4
+-- @tangra/angular-validators@1.1.4
+-- @tangra/kendo-grid-utils@1.0.25
+-- @types/googlemaps@3.30.16
+-- @types/jasmine@2.8.16
+-- @types/jasminewd2@2.0.6
+-- @types/node@8.9.5
+-- bootstrap@4.2.1
+-- codelyzer@4.5.0
+-- core-js@2.6.3
+-- google-maps-angular2@1.1.2
+-- jasmine-core@2.99.1
+-- jasmine-spec-reporter@4.2.1
+-- jquery@3.3.1
+-- karma@3.0.0
+-- karma-chrome-launcher@2.2.0
+-- karma-coverage-istanbul-reporter@2.0.4
+-- karma-jasmine@1.1.2
+-- karma-jasmine-html-reporter@0.2.2
+-- moment@2.24.0
+-- UNMET PEER DEPENDENCY popper.js@^1.14.6
+-- protractor@5.4.2
+-- UNMET PEER DEPENDENCY rxjs@6.2.2
+-- rxjs-compat@6.2.2
+-- ts-node@7.0.1
+-- tslib@1.9.3
+-- tslint@5.11.0
+-- typescript@3.1.6
+-- webpack-bundle-analyzer@3.0.3
`-- zone.js@0.8.29
```

Browser:

- Chrome (desktop) version 71

System:
- TypeScript version: 3.1.6
- Node version: 10.14.2  
- Platform:  Windows

**Comment by ipeshev** [comment]
Feature available in [latest dev version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by danielkaradachki** [comment]
Released in 3.7.0
